classes/phing/Phing.php

```diff
@@ -1719,7 +1688,7 @@ private static function setIni(): void
         self::$origIniSettings['short_open_tag'] = ini_set('short_open_tag', 'off');
         self::$origIniSettings['default_charset'] = ini_set('default_charset', 'iso-8859-1');
 
-        $mem_limit = (int) self::convertShorthand(ini_get('memory_limit'));
+        $mem_limit = intval(SizeHelper::fromHumanToBytes(ini_get('memory_limit')));
```

Comment:
Please use type cast instead of a function.

Reply:
Done